src/lib/util/util.js

```diff
@@ -152,10 +152,12 @@ class Util {
 			Object.freeze(source);
 			if (Array.isArray(source)) {
 				for (const value of source) Util.deepFreeze(value);
+			} else if (source instanceof Map) {
+				for (const value of source.values()) Util.deepFreeze(value);
```

Comment:
This freezes map values, but not keys. Is it unimportant that they be frozen?

Reply:
They're always primitives (strings, booleans, numbers...), they cannot be frozen.

Reply:
Do you mean in the context of SG, map keys will always be primitives? Because they can be objects in JS.